.github/workflows/lint.yml

```diff
@@ -28,10 +28,10 @@ jobs:
           fetch-depth: 0
       - uses: wagoid/commitlint-github-action@v3
 
-  mypy:
+  tox:
```

Comment:
I think `tox` might be misleading as that has a lot of unrelated test envs. Maybe just `lint` (`static-analysis` I guess sounds too verbose) and we can later chuck black in here as well? :)

```suggestion
  lint:
```

Reply:
Done. I called it "linters"